stylus/components/button.styl

```diff
@@ -76,34 +76,59 @@ $button
 $button--regular  // Deprecated
     @extend $button
 
-highlight = { name: 'highlight', bgColor: emerald, bdColor: emerald, focusColor: malachite, txtColor: white}
-danger = { name: 'danger', bgColor: pomegranate, bdColor: pomegranate, focusColor: monza, txtColor: white}
-alpha = { name: 'alpha', bgColor: transparent, bdColor: white, focusColor: scienceBlue, txtColor: white}
-secondary = { name: 'secondary', bgColor: white, bdColor: silver, focusColor: silver, txtColor: black}
-danger-outline = { name: 'danger-outline', bgColor: white, bdColor: yourPink, focusColor: yourPink, txtColor: pomegranate}
+highlightTheme = {
+    bgColor: emerald, bdColor: emerald, focusColor: malachite, txtColor: white
+}
+dangerTheme = {
+    bgColor: pomegranate, bdColor: pomegranate, focusColor: monza, txtColor: white
+}
+alphaTheme = {
+    bgColor: transparent, bdColor: white, focusColor: scienceBlue, txtColor: white
+}
+secondaryTheme = {
+    bgColor: white, bdColor: silver, focusColor: silver, txtColor: black
+}
+dangerOutlineTheme = {
+    bgColor: white, bdColor: yourPink, focusColor: yourPink, txtColor: pomegranate
+}
```

Comment:
(copy pasted from an outdated comment)

As we aim to be able to easy build ButtonText  (or _no-outline_ buttons) from themes, and knowing that in those buttons color are "inverted" what do you think about naming attributes something like:
```styl
highlight  = { name: 'highlight', mainColor: emerald, secondaryColor: emerald, activeColor: malachite, contrastColor: white }
```

Reply:
I kinda agree with @gregorylegarec on this. Could be a good time to do so.

Reply:
done, kudos for the names !